app/controllers/course/assessment/submission/answer/programming/files_controller.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+class Course::Assessment::Submission::Answer::Programming::FilesController < \
+  Course::Assessment::Submission::Answer::Programming::Controller
+  before_action :set_programming_answer
+  load_resource :file, class: Course::Assessment::Answer::ProgrammingFile.name,
```

Comment:
Shall we also add some abilities, and authorize the resource to ensure that not everyone can view such files?

Reply:
Added